docker/manylinux-entrypoint

```diff
@@ -4,6 +4,11 @@ set -eu
 
 if [ "${AUDITWHEEL_ARCH}" == "i686" ]; then
 	linux32 "$@"
+elif [ "${AUDITWHEEL_ARCH}" == "armv7l" ] && [ "$(uname -m)" != "armv7l" ]; then
+	if [ "$(linux32 uname -m)" == "armv8l" ]; then
+		export _PYTHON_HOST_PLATFORM="linux_armv7l"
+	fi
```

Comment:
This is really a hugly hack and should not be required for the image to be usable IMHO.